.github/workflows/release-python.yaml

```diff
@@ -0,0 +1,92 @@
+# Note that for this workflow to be triggered, the tag needs to be
+# created of the form `python/<version>+<buildinfo>`, where <buildinfo>
+# by convention is YYYYMMDD-<short-sha> (short SHA can be calculated
+# with `git rev-parse --short HEAD`). An example of a tag following
+# the convention that triggers automation would be
+# `python/7.3.33+20221123-8dfe8b9`.
+name: Release Python
+on:
+  push:
+    tags:
+      - python/*
+jobs:
+  release-python:
+    strategy:
+      matrix:
+        include:
+          - version: 3.11.1
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        # Only run for the Python version specified in the git tag.
+        #
+        # This if could be moved to the parent `job` section when it's
+        # supported by GitHub (https://github.com/community/community/discussions/37883)
+        if: startsWith(github.event.ref,  format('refs/tags/python/{0}+', matrix.version))
+        uses: actions/checkout@v3
+      - name: Build Python
+        # Only run for the Python version specified in the git tag.
+        #
+        # This if could be moved to the parent `job` section when it's
+        # supported by GitHub (https://github.com/community/community/discussions/37883)
+        if: startsWith(github.event.ref, format('refs/tags/python/{0}+', matrix.version))
+        run: make python/v${{ matrix.version }}
+      - name: Rename release artifacts
+        # Only run for the Python version specified in the git tag.
+        #
+        # This if could be moved to the parent `job` section when it's
+        # supported by GitHub (https://github.com/community/community/discussions/37883)
+        if: startsWith(github.event.ref, format('refs/tags/python/{0}+', matrix.version))
+        shell: bash
+        run: |
+          sudo mv python/build-output/python/v${{ matrix.version }}/bin/python{,-${{ matrix.version }}}.wasm
+      - name: Install zip
+        uses: montudor/action-zip@v1
```

Comment:
I'm pretty sure this will fail on the vmware-labs organization, given they have an allowlist of allowed actions. It's a very short list, and I don't remember seeing this on the list when I got the error when I tried to use a third party action in the past.

The workflow runner, does not come with a zip binary installed already?

Reply:
This action conveniently providers a "working-directory" property. Will try with bash. There should be zip.